stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/mega-menu.scss

```diff
@@ -189,12 +146,65 @@
   }
 }
 
+.global-header{
+  .mega-nav-option{
+    animation-fill-mode: backwards;
+    &.show-mega {
+      @extend %mega-animation;
+      -webkit-animation-fill-mode: forwards;
+      animation-fill-mode: forwards;
+  
+      > div{
+        -webkit-animation-delay: .3s;
+        animation-delay: .3s;
+        -webkit-animation-duration: .3s;
+        animation-duration: .3s;
+        -webkit-animation-fill-mode: forwards;
```

Comment:
@Ashish-srij We have created a mixin for the animation. Why are we not using that? Just use it simply in the following way:
Eg.
```
@include animation(transition-opacity forwards 0.3s 0.3s);
```
![image](https://user-images.githubusercontent.com/94124218/150992246-ca995346-358c-4afb-b09e-11e3e81fdbe1.png)